CODEOWNERS

```diff
@@ -0,0 +1,19 @@
+# Owners
+
+SPP Statisitcal Methods Library team
+
+* Neil Smith
+* Sanjeev Srithevan
+* Steve Gibbard
+* James Morgan
+* Jason bell
+* Kirsty Mackenzie
+
+# deployment related files
```

Comment:
Where did you get the information to add this? We don't want to mention the other repositories in this file.

Reply:
Removed :) I saw it in another example :)